src/controllers/expenseController.js

```diff
@@ -0,0 +1,90 @@
+const {
+  getExpenses,
+  getExpenseById,
+  create,
+  remove,
+  update,
+} = require('../services/expensesService');
+const userService = require('../services/userService');
+
+const getAll = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+
+  res.statusCode = 200;
+  res.send(getExpenses(userId, categories, from, to));
```

Comment:
Consider wrapping the service call `getExpenses` in a try-catch block to handle any potential errors that might occur during the execution.